db/config.cc

```diff
@@ -862,6 +862,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , large_memory_allocation_warning_threshold(this, "large_memory_allocation_warning_threshold", value_status::Used, size_t(1) << 20, "Warn about memory allocations above this size; set to zero to disable")
     , enable_deprecated_partitioners(this, "enable_deprecated_partitioners", value_status::Used, false, "Enable the byteordered and random partitioners. These partitioners are deprecated and will be removed in a future version.")
     , enable_keyspace_column_family_metrics(this, "enable_keyspace_column_family_metrics", value_status::Used, false, "Enable per keyspace and per column family metrics reporting")
+    , enable_node_table_metrics(this, "enable_node_table_metrics", value_status::Used, true, "Enable aggregated per node, per keyspace and per table metrics reporting, applicable if enable_keyspace_column_family_metrics is false")
```

Comment:
Please make it LiveUpdate (and dynamically enable/disable the metrics) so people don't have to do a rolling restart.

Reply:
Metrics are registered once on startup. The reason I've added this configuration is for a situation that something is so wrong, it would crash scylla (hopefully, never).

The metrics layer support changes in runtime. This is the reason the extra label  ```__per_table="node" ``` was added, so it would be easy to manipulate those metrics if needed.

Reply:
ok